emboss.md

```diff
@@ -0,0 +1,15 @@
+#Support Importing Emboss Files for Serialization/Deserialization Functions
```

Comment:
Can you put a space between the pounds and the title, ie. `# Support Importing`. That is the recommended style.